audformat/core/table.py

```diff
@@ -737,6 +766,190 @@ def set(
         for idx, data in values.items():
             self.columns[idx].set(data, index=index)
 
+    def update(
+            self,
+            others: typing.Union['Table', typing.Sequence['Table']],
+            *,
+            overwrite: bool = False,
+    ) -> 'Table':
+        r"""Update table with other table(s).
+
+        Table which calls ``update()`` must be assigned to a database.
+
+        Columns that are not yet part of the table will be added and
+        referenced schemes or raters are copied.
+        For overlapping columns, schemes and raters must match.
+        This also holds if table is assigned to a media or split.
```

Comment:
I would remove this sentence here, put it to the first table block, and change it to:
```
Table which calls ``update()`` must be assigned to a database.
For all tables media and split must match.
```

Reply:
done